### ISSUE TITLE
Invisible and strange texture

### ISSUE THREAD

**Issue by Otheucarvalho2** [ThirteenAG/GTAIV.EFLC.FusionFix#629]
### Game version this happens in

v1.2.0.59

### What area of the game

I don't really know what could be causing this I only have FF installed, I even uninstalled the game and installed it again, but I noticed that without FF the game is also that way

I believe it is related to the last update that the game had because it has been many months since I played and when I went to play again today it was like this

![image](https://github.com/user-attachments/assets/58dd4914-9c34-4bb0-901b-141b86d34c19)
![image](https://github.com/user-attachments/assets/3e81f63f-fef7-4ef3-9eec-e6789fe91067)
![image](https://github.com/user-attachments/assets/132f8782-51a7-4e62-a644-51013a712420)
![image](https://github.com/user-attachments/assets/3bc3e3a7-0c18-4431-ab8e-c15cb0161f1e)

apparently it's without collision too
![image](https://github.com/user-attachments/assets/6d74db08-51d6-471f-9322-c95cd0bdf683)

### What should happen

textures were supposed to appear here

### Checklist

- [X] Test in the [latest git build](https://ci.appveyor.com/api/projects/ThirteenAG/gtaiv-eflc-fusionfix/artifacts/GTAIV.EFLC.FusionFix.zip) in case it's already fixed.
- [X] [Search for other reports](https://github.com/ThirteenAG/GTAIV.EFLC.FusionFix/search?q=my+issue&type=issues) of the same issue.
- [X] Include logs or screenshots of issue.

**Comment by Adas8812** [comment]
Yeah I have the same issue. Also, this seems to happen in Bohan next to that garage where you have to blow it up with the truck. So now after leaving Bohan safehouse and turning right a bridge type texture is vissible across the street. I think this and Roman's cab firm texture bug are related because both areas were on fire /exploded and so did their textures/models change. 
Furthermore, this does not happen in TBOGT. I tried driving around those spots and It's fine.

**Comment by Otheucarvalho2** [comment]
> Yeah I have the same issue. Also, this seems to happen in Bohan next to that garage where you have to blow it up with the truck. So now after leaving Bohan safehouse and turning right a bridge type texture is vissible across the street. I think this and Roman's cab firm texture bug are related because both areas were on fire /exploded and so did their textures/models change. Furthermore, this does not happen in TBOGT. I tried driving around those spots and It's fine.

I believe it might be related, lad and tbogt don’t have this bug, and I always used only fusion to avoid breaking my save

**Comment by Otheucarvalho2** [comment]
> Are we sure this isn't save corruption?
> 
> Are we sure we started a new game after installing any mod that changes anything map related?

I really don’t know, it wasn’t like this before
